app/elements/socobo-ranking/socobo-ranking.html

```diff
@@ -542,7 +546,7 @@
          */
         _reload: function() {
           // notify user
-          this.notify("Reloading...");
+          this.notify("Ranking - Reloading...");
```

Comment:
Yes, but i could add a flag that The notification only appears if i hit The reload button.